src/lib/shell/streamer_zephyr.cpp

```diff
@@ -24,8 +24,8 @@
 
 #include <cassert>
 
-#include <shell/shell.h>
-#include <shell/shell_uart.h>
+#include <lib/shell/shell.h>
+#include <lib/shell/shell_uart.h>
```

Comment:
Neither of those files is in our tree, so this change is wrong.

Reply:
Fixed